jmix-pivottable/pivottable-flowui-kit/src/main/resources/META-INF/frontend/src/pivot-table/jmix-pivot-table.css

```diff
@@ -384,6 +386,10 @@ html[theme~="dark"] jmix-pivot-table {
         background-image: url('data:image/svg+xml;utf8,<svg width="24" height="24" viewBox="0 0 24 24" fill="%23ffffff" xmlns="http://www.w3.org/2000/svg"><path d="M6.780692 9.39493c-.43905-.393718-1.11414-.356968-1.507858.082082-.393716.43905-.356966 1.114141.082084 1.507858l5.864118 5.381134c.405651.363767 1.020124.363767 1.425775 0l6.000272-5.503229c.43905-.393717.4758-1.068807.082083-1.507857-.393717-.43905-1.068808-.4758-1.507858-.082083l-5.287384 4.863949-5.151232-4.741855z" opacity="0.5"/></svg>');
     }
 
+    select option {
```

Comment:
Unfortunately, the `option` element allows you to style only `background-color` and `color` attributes: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#styling_with_css